app/api/v1/endpoints/ai_consensus.py

```
@@ -0,0 +1,816 @@
+        # Get comprehensive AI status from existing service
+        result = await ai_consensus_service.health_check()
+        
+        # Add user-specific information
+        user_weights = await master_controller.get_user_ai_model_weights(str(current_user.id))
+        result["user_config"] = user_weights
+        
+        # Add emergency status
+        emergency_status = await emergency_manager.get_emergency_status(str(current_user.id))
+        result["emergency_status"] = emergency_status
+        
+        return result
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)